test/keystore/InMemoryKeystore.test.ts

```diff
@@ -457,7 +457,13 @@ describe('InMemoryKeystore', () => {
       const shuffled = [...timestamps].sort(() => Math.random() - 0.5)
 
       await Promise.all(
-        shuffled.map((createdAt) => {
+        shuffled.map(async (createdAt) => {
+          let keys = await PrivateKeyBundleV1.generate(newWallet())
+
+          const recipient = SignedPublicKeyBundle.fromLegacyBundle(
```

Comment:
I might be dumb, but it's not obvious to me how this change tests deterministic v2 conversation topics. Should we right a dedicated test with something really targeted? e.g.
```
generate Alice
generate Bob
alice generates 100 invites for bob
confirm all 100 invites have the same topicKey and same topic
confirm the invites differ in timestamp
```

Reply:
This test change was just because it was failing due to the implementation changes. I added a new test to make sure we're using the same topic in https://github.com/xmtp/xmtp-js/pull/326/commits/551c31e2e0a18e8aab1767e62230f1aa61467e28.

I wasn't sure how to get the topic key off the conversation objects we get back from the keystore. Is that an internal thing?